### PR TITLE
Release/0.15.0

### PR DESCRIPTION
**Pull request by nghazali**

### CODE REVIEW

File: src/main/charts/bitbucket/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: bitbucket
 description: A chart for installing Bitbucket Data Center on Kubernetes
 type: application
-version: 0.14.0
+version: 0.15.0
 appVersion: 7.14.1-jdk11
```
Comment (nanux) [inline src/main/charts/bitbucket/Chart.yaml:6]: The latest Bitbucket version is `7.15.1`, maybe we can upgrade it when we are doing changes anyway.
cc @wkritzinger-atlassian 
*note*: changing the version will update the `README.md` file for the chart and also require updated expected output test resource.

File: src/main/charts/bitbucket/Chart.yaml [hunk 2]
```diff
@@ -19,8 +19,35 @@ sources:
...
+      links:
+      - name: Github PR
+        url: https://github.com/atlassian-labs/data-center-helm-charts/pull/226
+    - kind: changed
+      description: Renamed the 'master' branch to 'main'
+      links:
+      - name: Github PR
+        url: https://github.com/atlassian-labs/data-center-helm-charts/pull/232
+    - kind: changed
+      description: Update eks cluster yaml
```
Comment (nanux) [inline src/main/charts/bitbucket/Chart.yaml:38]: `Update EKS cluster yaml example`

File: src/main/charts/confluence/Chart.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: confluence
 description: A chart for installing Confluence Data Center on Kubernetes
 type: application
-version: 0.14.0
+version: 0.15.0
 appVersion: 7.12.3-jdk11
```
Comment (nanux) [inline src/main/charts/confluence/Chart.yaml:6]: `7.12.3-jdk11` is the latest released version, we can update it
cc @badgersow

File: src/main/charts/confluence/Chart.yaml [hunk 4]
```diff
@@ -19,8 +19,35 @@ sources:
...
+      links:
+      - name: Github PR
+        url: https://github.com/atlassian-labs/data-center-helm-charts/pull/218
+    - kind: changed
+      description: Renamed the 'master' branch to 'main'
+      links:
+      - name: Github PR
+        url: https://github.com/atlassian-labs/data-center-helm-charts/pull/232
+    - kind: changed
+      description: Update eks cluster yaml
```
Comment (nanux) [inline src/main/charts/confluence/Chart.yaml:38]: The same as above.

File: src/main/charts/confluence/Chart.yaml (hunk 3 above)
Comment (nghazali) [reply src/main/charts/confluence/Chart.yaml:6]: `7.12.4-jdk11` is the latest version

Comment (nanux) [reply src/main/charts/confluence/Chart.yaml:6]: 🤦 thinking one number, writing different number :) Yes, I've meant 7.12.4

**Comment by nanux** [comment]
https://server-syd-bamboo.internal.atlassian.com/browse/DCNG-K8SHELMTEST281 is green as well.

**Review by nanux — APPROVED** [review]

**Pull request merged**